packages/web-frontend/src/containers/MapOverlayBar/TitleAndDatePicker.js

```diff
@@ -54,6 +62,15 @@ const Pin = styled(PinBase)`
   margin: 3px 12px 0 7px;
 `;
 
+const CircularProgress = styled(MuiCircularProgress)`
+  color: #2196f3;
+`;
+
+const Skeleton = styled(MuiSkeleton)`
+  margin-left: ${({ ml = 0 }) => ml}px;
+  margin-top: ${({ mt = 0 }) => mt}px;
```

Comment:
```suggestion
  margin-top: ${({ mt = 0 }) => mt}px;
  background: none;
```
To make the fading effect from the design, you can turn off the background color on the skeleton.